contentcuration/contentcuration/frontend/shared/utils.js

```diff
@@ -335,7 +342,20 @@ export async function generatePdf(
             );
           })
         );
-      } else if (!node.childElementCount && node.innerText) {
+      } else if (isNonUnicode) {
+        promises.push(
+          html2canvas(node).then(canvas => {
+            doc.addImage(
+              canvas.toDataURL(),
+              'PNG',
+              scale * (x - 8), // Account for padding
```

Comment:
Is this fixed, or variable depending on the padding attributes?

Reply:
It seems to be fixed since the element I tested on doesn't have any padding